texts/epn_paper.md

```diff
@@ -32,7 +32,10 @@ After several discussions we committed to developing a community platform for on
 <!-- #### In thinking about online events we had to revisit many assumptions about conferences. -->
 
 Once we started brainstorming online events, we realized, that despite those are similar to regular scientific meetings, a lot of our assumptions about conferences do not hold anymore.
-For example, the interactions between the speaker and the audience as well as among the audience themselves in an online event are quite different and much less spontaneous.
+For example, the interactions between the speaker and the audience as well as among the audience themselves in an online event are quite different and much less spontaneous. 
+As a speaker, for example, gauging the audience's reaction is much harder due to the lack of visual and audio contact -- most participants will have their audio muted and their video turned off. 
```

Comment:
The repetition of *for example* reads odd.

Suggestion:
As a speaker, for instance, gauging the audience's reaction is much harder due to the lack...

Reply:
```suggestion
For example, the interactions between the speaker and the audience as well as among the audience themselves in an online event are quite different and much less spontaneous. 
As a speaker, for instance, gauging the audience's reaction is much harder due to the lack of visual and audio contact -- most participants will have their audio muted and their video turned off. 
```

Reply:
Thank for noticing that! I now realized that the transition from the previous sentence to these two is a bit weird. The previous bit talks about assumptions on conferences; Maybe that needs to change?